readthedocs/templates/support/index.html

```diff
@@ -0,0 +1,142 @@
+you can file bug reports on our <a href="https://github.com/readthedocs/readthedocs.org/issues">GitHub issue tracker</a>.
+You can also <a href="http://docs.readthedocs.io/page/contribute.html">contribute</a> to Read the Docs,
+as the code is open source.
+</p>
+
+<h2 id="commercial-support">Commercial Support</h2>
+
+<p>
+We offer commercial support for Read the Docs
+and commercial hosting with <a href="https://readthedocs.com">Read the Docs for Business</a>.
+We have a dedicated team that responds to support requests during business hours.
+</p>
+
+<h2 id="user-support">User Support</h2>
+
+<p>
+If you need a specific request for your project or account,
+like more resources, change of the project's slug or username.
+</p>
+
+{% if not SUPPORT_FORM_ENDPOINT %}
+
+<p>
+Please send an email to <a href="mailto:{{SUPPORT_EMAIL }}">{{ SUPPORT_EMAIL }}</a>
+and we will get back to you as soon as possible.
+</p>
+
+{% else %}
+
+<p>
+Fill out the form below,
+and we will get back to you as soon as possible.
+</p>
+
+<form
+  method="POST"
+  name="support-request"
+  action="{{ SUPPORT_FORM_ENDPOINT }}"
+  enctype="multipart/form-data"
+  accept-charset="utf-8"
+>
+    <label for="name">Name</label>
+    <input type="text" name="name" {% if request.user.is_authenticated %} value="{{ request.user.first_name }} {{ request.user.last_name }}" {% endif %}>
+    <p></p>
+  </div>
+
+  <div>
+    <label for="email">Email</label>
+    <input type="email" name="email"  {% if request.user.is_authenticated %} value="{{ request.user.email }}" {% endif %}>
+    <p></p>
+  </div>
+
+  <div>
+    <label for="body">Explanation of issue</label>
+    <textarea name="body" required></textarea>
+    <p>
+       Please provide as much detail as possible.
+     </p>
+  </div>
+
+  <div>
+    <label for="url">URL</label>
+    <input type="text" name="url"></textarea>
+    <p>
+       Is there a specific page this happened?
+    </p>
+  </div>
+
+  <div>
+    <label for="file">Screenshot or additional file</label>
+    <input type="file" name="attachment">
+    <p>
+       Anything else that would help us solve this issue?
+    </p>
+  </div>
+
+  <div>
+    <label for="level">Severity Level</label>
+        {% if request.user.gold.exists or request.user.goldonce.exists %}
+        <select>
+            <option value="" selected>Select level</option>
+        {% else %}
+        <select disabled>
+            <option value="" selected>Field disabled</option>
+        {% endif %}
+            <option value="low">Low</option>
+            <option value="medium">Medium</option>
+            <option value="high">High</option>
+        </select>
+        {% if request.user.gold.exists or request.user.goldonce.exists %}
+        <p> Please rate the severity of this event. </p>
+        {% else %}
+        <p>This option is only enabled for <a href="{% url "gold_subscription" %}">Gold users</a>.</p>
+        {% endif %}
```

Comment:
This should be enabled for commercial plans too. Is there a good way to do that here and not require an overridden template?

Reply:
Yea, I want an abstraction that works for both .org & .com around this. I don't think we have one, and didn't know if it was in scope for this PR :)

I haven't ported this to commercial yet, was planning to do that today and figure out a plan.

Reply:
Good point, yeah we might want some more information on this page in commercial.

I suppose we could also unconditionally ask for the severity level and maybe avoid overriding the template. We wouldn't need to really use this information for anything but data collection, except for upper tier plans where response time varies by severity levels.